### PR TITLE
Fix Misalignment of FloatingActionButton and BottomNavBar is Blank

### LINKED ISSUE #2984: Misalignment of `FloatingActionButton`
**Describe the bug**
The `FloatingActionButton` stays above the app is newly installed.

**Expected behavior**
Align with the top of `BottomNavigationBar`
**Steps to reproduce the behavior:**
* Fresh install the app

**Screenshots**
Screenshot_2022-08-27-14-23-41-10_924282dd390cc14aae1af1e29436bb16

**Environment**
- Version of Kiwix Android : 3.7.0
- Device : Realme
- OS version : 11

### LINKED ISSUE #3069: `BottomNavBar` is Blank
**Describe the bug**
When we scroll the Zim list in the libarary tab, in the BottomNavBar it shows a blank screen. 
**Expected behavior**
Hides the Bottom Nav Bar and show the Zim item book view
**Steps to reproduce the behavior:**
1. Open the App
2. Scroll the App
3. Observe the Bottom Nav Bar

**Screenshots**
Screenshot_2022-09-06-02-37-48-80_924282dd390cc14aae1af1e29436bb16

**Environment**
- Version of Kiwix Android: 3.7.0
- Device: realme
- OS version: 11

**Logs**

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #2984
Fixes #3069 

Fix Misalignment of FloatingActionButton and BottomNavBar is Blank.
### Whats the Issue
We Are Setting margin to parent fragment in KiwixReaderFragment
```
 private fun setFragmentContainerBottomMarginToSizeOfNavBar() {
    val actionBarHeight = context?.getAttribute(android.R.attr.actionBarSize)
    if (actionBarHeight != null) {
      setParentFragmentsBottomMarginTo(
        complexToDimensionPixelSize(
          actionBarHeight,
          resources.displayMetrics
        )
      )
    }
  }
```
on first run we take permission of storage and then we ask ``` MANAGE_EXTERNAL_PERMISSION ``` from user (in android 11 and above) so ```onPause``` method of ```KiwixReaderFragment``` not called where we set margin 0 to parent fragment that's why this blank space is visible on first run.

### Fix:
Removed the extra margin from parent fragment in ```LocalLibraryFragment``` and ```OnlineFragment``` and make Fragment extension function to reusability of ```setParentFragmentsBottomMarginTo``` method.

### CODE REVIEW

**Comment by kelson42** [comment]
@MohitMaliFtechiz Should this PR be reviewed?

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliFtechiz Should this PR be reviewed?

Yes just requested were waiting for CI to get green.

**Review by gouri-panda — DISMISSED** [review]
Looks good code-wise, but I think maybe we should add a little more bottom margin on top of the bottom bar. I'll let @kelson42 review the UI.

**Comment by kelson42** [comment]
May I ask for a screenshot?

**Comment by gouri-panda** [comment]
@kelson42 Sure 
Screenshot_2022-09-10-00-42-51-44_924282dd390cc14aae1af1e29436bb16 If you need APK i can give it you.

**Comment by kelson42** [comment]
@gouri-panda Thank you for the screenshot. I understand now your remark about the bottom margin. I agree, it shoukd slightly bigger and be the same at the right margin.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Can you please fix this small glitch and provide a new screenshot?

**Comment by MohitMaliFtechiz** [comment]
Hi @kelson42 
here is updated screenshot of fab buttton.
Screenshot_2022-09-12-13-14-01-037_org kiwix kiwixmobile

**Pull request merged**
